packages/liquidsoap-core/liquidsoap-core.2.3.0/opam

```diff
@@ -0,0 +1,135 @@
+opam-version: "2.0"
+synopsis: "Liquidsoap core library and binary"
+maintainer: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+license: "GPL-2.0-or-later"
+homepage: "https://github.com/savonet/liquidsoap"
+bug-reports: "https://github.com/savonet/liquidsoap/issues"
+depends: [
+  "dune" {>= "3.6"}
+  "ocaml" {>= "4.14"}
```

Comment:
```suggestion
  "ocaml" {>= "4.14" & < "5"}
```
Due to
```
#=== ERROR while compiling liquidsoap-core.2.3.0 ==============================#
# context              2.4.0~alpha1~dev | linux/x86_64 | ocaml-base-compiler.5.3.0~beta1 | pinned(https://github.com/savonet/liquidsoap-release-assets/releases/download/v2.3.0/liquidsoap-2.3.0.tar.gz)
# path                 ~/.opam/5.3~beta1/.opam-switch/build/liquidsoap-core.2.3.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p liquidsoap-core -j 31 --promote-install-files=false @install
# exit-code            1
# env-file             ~/.opam/log/liquidsoap-core-7-6a53a8.env
# output-file          ~/.opam/log/liquidsoap-core-7-6a53a8.out
### output ###
# (cd _build/.sandbox/d376fcf076e65f5c9a3c9a668b29025b/default && .ppx/6fd25187b0e70872735b78e12e8bdfcf/ppx.exe --cookie 'library-name="liquidsoap_core"' -o src/core/operators/compress.pp.ml --impl src/core/operators/compress.ml -corrected-suffix .ppx-corrected -diff-cmd - -dump-ast)
# File "src/core/operators/compress.ml", line 36, characters 16-22:
# 36 |     val mutable effect = None
#                      ^^^^^^
# Error: Syntax error
# (cd _build/.sandbox/9877cdff56c1674a27ba10efa2ff215e/default && .ppx/6fd25187b0e70872735b78e12e8bdfcf/ppx.exe --cookie 'library-name="liquidsoap_core"' -o src/core/operators/echo.pp.ml --impl src/core/operators/echo.ml -corrected-suffix .ppx-corrected -diff-cmd - -dump-ast)
# File "src/core/operators/echo.ml", line 35, characters 16-22:
# 35 |     val mutable effect = None
#                      ^^^^^^
# Error: Syntax error
# (cd _build/.sandbox/afeb96eee8e65c3aaef262311711c031/default && .ppx/6fd25187b0e70872735b78e12e8bdfcf/ppx.exe --cookie 'library-name="liquidsoap_core"' -o src/core/operators/video_effects.pp.ml --impl src/core/operators/video_effects.ml -corrected-suffix .ppx-corrected -diff-cmd - -dump-ast)
# File "src/core/operators/video_effects.ml", line 28, characters 18-24:
# 28 | let cached_effect effect =
#                        ^^^^^^
# Error: Syntax error
```
@toots are you ok with this or you would prefer to rename the variable?

Reply:
Yeah that's good. We don't recommend using OCaml 5 for now because of: https://github.com/ocaml/ocaml/issues/13123